subsys/bluetooth/controller/Kconfig

```
@@ -353,6 +353,20 @@ config BT_CTLR_CONN_RSSI
 	help
 	  Enable connection RSSI measurement.
 
+config BT_CTLR_CONN_RANDOM_FORCE
+	bool "Enable random forced sceduling for peripheral on missed anchor point"
```

```suggestion
	bool "Enable random forced scheduling for peripheral on missed anchor point"
```

PR CI does not flag misspells, please correct sceduling